src/NHapi.Model.V231/Datatype/XPN.cs

```diff
@@ -9,14 +9,14 @@ namespace NHapi.Model.V231.Datatype
 
 ///<summary>
 /// <p>The HL7 XPN (extended person name) data type.  Consists of the following components: </p><ol>
-/// <li>family+last name (FN)</li>
-/// <li>given name (ST)</li>
-/// <li>middle initial or name (ST)</li>
-/// <li>suffix (e.g., JR or III) (ST)</li>
-/// <li>prefix (e.g., DR) (ST)</li>
-/// <li>degree (e.g., MD) (IS)</li>
-/// <li>name type code (ID)</li>
-/// <li>Name Representation code (ID)</li>
+/// <li>PID-5.1 :family+last name (FN)</li>
```

Comment:
Data types can be used by any segment type not just `PID` so it doesn't make sense to include this in the xml comments.